### PR TITLE
fix: Change "Apps" to "Website Generator" and other improvements

### LINKED ISSUE #5786: Event Dashboard: Change "Apps" to "Website Generator" and other improvements
Currently the Android app is not functional. Therefore we should not feature it on the event dashboard. We can also enhance the area and the dashboard with a few smaller changes as follows:
* Change box heading "Event sponsors" to "Sponsors"
* Change box heading "Apps" to "Website Generator"
* Add the following text: You would like to have a dedicated event website with all details, sponsors listing, speakers and sessions and the entire schedule? Simply use the website generator. It transfers the data of the event via an API to our website tool and creates a website for you. You can download the entire site as a zip file and upload it to your own web hosting service. No database set up or framework required. This feature is still in Alpha stage, but is already used actively by a number of events. If you press the below button a tab will open where your website is generated.
* Below the text show a button "Create Website" (change the current button "Generate Web App"). Use a nice icon and use similar style to the download button for tickets.

Screenshot from 2020-11-27 18-20-01

### PR DESCRIPTION
**Pull request by sachinchauhan2889**
Fixes #5786 

 __screenshots__

dashboard 2

#### Checklist

- [X] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [X] My branch is up-to-date with the Upstream `development` branch.
- [X] The acceptance, integration, unit tests and linter pass locally with my changes 
- [X] I have added tests that prove my fix is effective or that my feature works
- [X] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Comment by iamareebjamal** [comment]
Fix the build please

**Review by iamareebjamal — APPROVED** [review]

**Comment by mariobehling** [comment]
Thanks, could you reduce the spacing of the text and buttons on the top and bottom please to match other similar areas. Also please right-align the "Create Website" button.

**Comment by mariobehling** [comment]
Please see example below.

Screenshot from 2020-11-27 20-56-10

**Comment by iamareebjamal** [comment]
Please update screenshot

**Comment by sachinchauhan2889** [comment]
generator

**Comment by iamareebjamal** [comment]
Text still has a lot of space above it. Compare to the image by Mario

**Comment by sachinchauhan2889** [comment]
generator new

File: app/templates/components/events/view/overview/event-apps.hbs [hunk 1]
```diff
@@ -1,13 +1,11 @@
...
 <div class="content ui stackable grid">
-  <div class="row">
-    <div class="column eight wide center aligned">
-      <button class="ui button">{{t 'Generate Android App'}}</button>
-    </div>
-    <div class="column eight wide center aligned">
-      <a href="{{this.webAppGeneratorUrl}}" target="_blank" class="ui button" rel="noopener noreferrer">{{t 'Generate Web App'}}</a>
-    </div>
+  <div class="row pt-0">
+    {{t 'You would like to have a dedicated event website with all details, sponsors listing, speakers and sessions and the entire schedule? Simply use the website generator. It transfers the data of the event via an API to our website tool and creates a website for you. You can download the entire site as a zip file and upload it to your own web hosting service. No database set up or framework required. This feature is still in Alpha stage, but is already used actively by a number of events. If you press the below button a tab will open where your website is generated.'}}
```
Comment (iamareebjamal) [inline app/templates/components/events/view/overview/event-apps.hbs:6]: Please break the sentences into multiple t statements. This way, translators don't have to retranslate the whole paragraph if a single word changes

**Review by mariobehling — DISMISSED** [review]

**Pull request merged**